### PR TITLE
[zh-cn] Create translation for `max` attribute

### PR DESCRIPTION
**Pull request by MaxtuneLee**
### Description
add translation for `max` attribute

### Motivation
`max` attribute is frequently used in daily development, it would be better if it has Chinese translation

### Related issues and pull requests
Relates to https://github.com/mdn/translated-content/issues/17632

### CODE REVIEW

File: files/zh-cn/web/html/attributes/max/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: HTML 属性：max
+slug: Web/HTML/Attributes/max
+l10n:
+  sourceCommit: fcbe69e21d85708248ff60021b1937817844f51c
+---
+
+{{HTMLSidebar}}
+
+**`max`** 属性定义了输入字段所能接受的最大值。如果元素的 [`value`](/zh-CN/docs/Web/HTML/Element/input#value) 大于此值，则该元素将不能通过[校验](/zh-CN/docs/Learn/Forms/Form_validation)。`max` 属性值必须大于或等于 [`min`](/zh-CN/docs/Web/HTML/Attributes/min) 属性的值。如果存在 `max` 属性但未指定或无效，则 `max` 值不会被应用。如果 `max` 属性有效，并且非空 `value` 值大于 `max` 属性所允许的最大值，则约束验证将阻止表单提交。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:10]:
```suggestion
**`max`** 属性定义了输入字段所能接受的最大值。如果元素的 [`value`](/zh-CN/docs/Web/HTML/Element/input#value) 大于此值，则该元素将不能通过[校验](/zh-CN/docs/Learn/Forms/Form_validation)。此值必须大于等于 [`min`](/zh-CN/docs/Web/HTML/Attributes/min) 属性的值。如果存在 `max` 属性但未指定或无效，则 `max` 值不会被应用。如果 `max` 属性有效，并且非空值大于 `max` 属性所允许的最大值，则约束验证将阻止表单提交。
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+`max` 属性适用于数值类型的输入字段，包括 {{HTMLElement("input/date", "date")}}、{{HTMLElement("input/month", "month")}}、{{HTMLElement("input/week", "week")}}、{{HTMLElement("input/time", "time")}}、{{HTMLElement("input/datetime-local", "datetime-local")}}、{{HTMLElement("input/number", "number")}} 和 {{HTMLElement("input/range", "range")}} 类型，以及 {{htmlelement('progress')}} 和 {{htmlelement('meter')}} 元素。`max` 属性是一个数值，用于指定表单控件被认为有效的最大正值。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:12]:
```suggestion
max 属性适用于数值类型的输入，包括 {{HTMLElement("input/date", "date")}}、{{HTMLElement("input/month", "month")}}、{{HTMLElement("input/week", "week")}}、{{HTMLElement("input/time", "time")}}、{{HTMLElement("input/datetime-local", "datetime-local")}}、{{HTMLElement("input/number", "number")}} 和 {{HTMLElement("input/range", "range")}} 类型，以及 {{htmlelement('progress')}} 和 {{htmlelement('meter')}} 元素。它是一个数值，用于指定表单控件被认为有效的最大正值。
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        <code>&#x3C;input type="range" min="60" step="5" max="100"></code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+> [!NOTE]
+> 当用户输入的数据不符合设置的最大值时，该值在约束验证中被视为无效，并会匹配 {{cssxref(':invalid')}} 和 {{cssxref(':out-of-range')}} 伪类。
+
+请参阅[约束验证](/zh-CN/docs/Web/HTML/Constraint_validation) 和 {{domxref("ValidityState.rangeOverflow", "rangeOverflow")}} 了解更多信息。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:81]:
```suggestion
请参阅[客户端验证](/zh-CN/docs/Web/HTML/Constraint_validation)和 {{domxref("ValidityState.rangeOverflow", "rangeOverflow")}} 了解更多信息。
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+对于 {{htmlelement('progress')}} 元素，`max` 属性描述了 `progress` 元素所表示的任务一共需要完成多少工作。如果 `max` 属性存在，必须具有大于零的值，并且是有效的浮点数。对于 {{htmlelement('meter')}} 元素，`max` 属性定义了测量范围的上限值。此值必须大于最小值（如果指定了[`min`](/zh-CN/docs/Web/HTML/Attributes/min) 属性）。在这两种情况下，如果省略，默认值为 1。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:83]:
```suggestion
对于 {{htmlelement('progress')}} 元素，`max` 属性描述了 `progress` 元素所表示的任务一共需要完成多少工作。如果该属性存在，必须具有大于零的值，并且是有效的浮点数。对于 {{htmlelement('meter')}} 元素，`max` 属性定义了测量范围的上限值。此值必须大于最小值（如果指定了[`min`](/zh-CN/docs/Web/HTML/Attributes/min) 属性）。在这两种情况下，如果省略，默认值为 1。
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        <code
+          >&#x3C;meter id="fuel" min="0" max="100" low="33" high="66"
+          optimum="80" value="40"> at 40/100&#x3C;/meter></code
+        >
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## 无障碍访问注意事项
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:122]: 小标题的翻译请参考[术语表](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#%E6%A0%87%E9%A2%98%E8%A1%A8%E6%A0%BC)

```suggestion
## 无障碍考虑
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+开发者需要提供指引，帮助用户了解如何填写表单并使用各个表单控件。标明任何必填和可选的输入项、数据格式以及其他相关信息。在使用 `max` 属性时，确保用户理解这个最大值要求。可以通过 {{htmlelement('label')}} 提供说明。如果需要在标签之外提供上述说明以实现更灵活的布局设计，请考虑使用 [`aria-labelledby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) 或 [`aria-describedby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:124]:
```suggestion
提供帮助用户了解如何填写表单并使用各个表单控件的指引。标明任何必填和可选的输入项、数据格式以及其他相关信息。在使用 `max` 属性时，确保用户理解最大值的要求。可以通过 {{htmlelement('label')}} 提供说明。如果需要在标签之外提供上述说明以实现更灵活的布局设计，请考虑使用 [`aria-labelledby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) 或 [`aria-describedby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)。
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 另见
+
+- [`step`](/zh-CN/docs/Web/HTML/Attributes/step)
+- [`min`](/zh-CN/docs/Web/HTML/Attributes/min)
+- 其他 meter 属性：[`low`](/zh-CN/docs/Web/HTML/Attributes/low), [`high`](/zh-CN/docs/Web/HTML/Attributes/high), [`optimum`](/zh-CN/docs/Web/HTML/Attributes/optimum)
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:138]:
```suggestion
- 其他 meter 属性：[`low`](/zh-CN/docs/Web/HTML/Attributes/low)、[`high`](/zh-CN/docs/Web/HTML/Attributes/high)、[`optimum`](/zh-CN/docs/Web/HTML/Attributes/optimum)
```

File: files/zh-cn/web/html/attributes/max/index.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+- [约束验证](/zh-CN/docs/Web/HTML/Constraint_validation)
+- [表单校验](/zh-CN/docs/Learn/Forms/Form_validation)
+- {{domxref('validityState.rangeOverflow')}}
+- {{cssxref(':out-of-range')}}
+- {{htmlelement('input')}}
+- {{HTMLElement("input/date", "date")}}, {{HTMLElement("input/month", "month")}}, {{HTMLElement("input/week", "week")}}, {{HTMLElement("input/time", "time")}}, {{HTMLElement("input/datetime-local", "datetime-local")}}, {{HTMLElement("input/number", "number")}} 和 {{HTMLElement("input/range", "range")}} 类型，以及 {{htmlelement('meter')}}
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/max/index.md:144]:
```suggestion
- {{HTMLElement("input/date", "date")}}、{{HTMLElement("input/month", "month")}}、{{HTMLElement("input/week", "week")}}、{{HTMLElement("input/time", "time")}}、{{HTMLElement("input/datetime-local", "datetime-local")}}、{{HTMLElement("input/number", "number")}} 和 {{HTMLElement("input/range", "range")}} 类型，以及 {{htmlelement('meter')}}
```

**Review by yin1999 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/html/attributes/max/index.md (hunk 5 above)
Comment (MaxtuneLee) [reply files/zh-cn/web/html/attributes/max/index.md:122]: 了解，感谢提醒

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you @MaxtuneLee.

**Pull request merged**